paasta_tools/mesos_tools.py

```diff
@@ -137,6 +137,17 @@ def filter_not_running_tasks(tasks):
     return [task for task in tasks if task['state'] != 'TASK_RUNNING']
 
 
+def get_running_task_ids_from_mesos_slave():
```

Comment:
AFAICT this is identical to `[task['id'] for task in get_running_tasks_from_active_frameworks('')]`.